grafana/provisioning/datasources/postgres.yml

```diff
@@ -0,0 +1,17 @@
+apiVersion: 1
+
+datasources:
+  - name: Postgres
+    type: postgres
+    url: postgres
+    database: jarbas
+    user: jarbas
+    secureJsonData:
+      password: "mysecretpassword"
+    jsonData:
+      sslmode: "disable" # disable/require/verify-ca/verify-full
+      maxOpenConns: 0         # Grafana v5.4+
+      maxIdleConns: 2         # Grafana v5.4+
+      connMaxLifetime: 14400  # Grafana v5.4+
+      postgresVersion: 904 # 903=9.3, 904=9.4, 905=9.5, 906=9.6, 1000=10
+      timescaledb: false
```

Comment:
This YAML will provide out Postgres datasource.

Particularly, I haven't used for anything, but you can build dashboards about reimbursements with this datasource.